components/gallery/GalleryItemAction/GalleryItemActionType/GalleryItemBuy.vue

```diff
@@ -102,6 +102,9 @@ const onClickShoppingCart = () => {
 :deep(.button-height) {
   height: 55px !important;
 }
+.square-button-width {
+  width: 55px;
+}
```

Comment:
I would use px-4 instead (even if it's not square square)

Reply:
`size="large"` does not work well because the 55px height is applied on different elements. I have replaced it with `px-4` as you mentioned.

Reply:
> `size="large"` does not work well because the 55px height is applied on different elements. I have replaced it with `px-4` as you mentioned.

Have you tried to setup `size="large"` on the 2 `NeoButton` it was working on my side

Reply:
Yes, I have tried. It did not work as well.
<img width="653" alt="image" src="https://github.com/kodadot/nft-gallery/assets/31397967/7ea8c2ab-949f-4fc2-99ed-7175a6ff7cc0">
<img width="242" alt="image" src="https://github.com/kodadot/nft-gallery/assets/31397967/7bddbf1c-2ade-4251-9692-7e1dad46b17b">

Reply:
1px different ;(

Reply:
continue in #7957